code/style.css

```
@@ -1,5 +1,270 @@
-/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
+html {
+  height: 100%;
+  width: full-width;
+}
 
+head {
+  padding: 30px;
+  margin-bottom: 50px;
+  margin-top: 5px;
+  margin-left: 5px;
+  margin-right: 5px;
+}
 body {
-  background-color: lightblue;
+  height: 100%;
+  font-family: "Montserrat", sans-serif;
+  background-color: white;
+  margin-top: 95px;
+}
+
+h1 {
+  font-weight: bold;
+}
+
+h2 {
+  font-weight: bold;
+}
+
+.logo {
+  float: right;
+}
+.container2 {
+  position: absolute;
+  top: 0;
+  clear: none;
+  overflow: hidden;
+  background-color: white;
+}
+nav {
+  float: right;
+  position: absolute;
+  top: 0;
+  right: 30%;
+  padding: 30px;
+  overflow: hidden;
+  background-color: white;
+  height: 80px;
+  word-spacing: 5px;
+  margin-left: 50px;
+  margin-bottom: 50px;
+  text-transform: uppercase;
+}
```

With flexbox instead of an absolute position, you wouldn't need as many css properties and you could remove the `logo` class. Maybe try with the following:
```suggestion
.container2 {
  display: flex;
  justify-content: space-between;
  align-items: center;
  background-color: white;
}

nav {
  padding: 30px;
  background-color: white;
  word-spacing: 5px;
  text-transform: uppercase;
}
```